rundetection/rules/common_rules.py

```diff
@@ -66,6 +66,11 @@ def verify(self, job_request: JobRequest) -> None:
         job_request.additional_values["slice_wavs"] = self._value
 
 
+class SansPhiLimits(Rule[str]):
```

Comment:
I just notice that this rule and the 2 above are missing a class doc string. The method one should be inherited, but it would be nice to have a little explanation

Reply:
Also this is technically out of scope, but I _believe_ there is an exceptions module, can you move the exception above as well, if not too annoying